index.html

```diff
@@ -812,31 +721,31 @@ <h3>
           Release wake lock algorithm
         </h3>
         <p>
-          To <dfn>release a wake lock</dfn> for a given {{WakeLock}} object
-          |lock:WakeLock|, run these steps <a>in parallel</a>:
+          To <dfn>release a wake lock</dfn> for a given |lockPromise:Promise| and
+          |type:WakeLockType|, run these steps <a>in parallel</a>:
         </p>
         <ol class="algorithm" data-link-for="WakeLock">
-          <li>Let |type:WakeLockType| be |lock|'s <a>type</a>.
+          <li>Reject |lockPromise| with an "{{AbortError}}" {{DOMException}} and
```

Comment:
Assuming everything went fine in a call to `WakeLock.request()`, I'd assume the promise it returns would have been resolved, in which case we can't reject it here (we could reach this step after that happens by either canceling the request via `AbortController`, or the lock could be automatically released by the user agent due to a time limit or something).

Reply:
Handled